tests/vmcheck/test-layering-rpmdb.sh

```diff
@@ -127,48 +127,3 @@ if vm_rpmostree upgrade; then
   assert_not_reached "upgrade succeeded but new base has conflicting pkg foo"
 fi
 echo "ok can't upgrade with conflicting layered pkg"
-
-vm_rpmostree cleanup -p
-
-check_bloop_color() {
```

Comment:
The colouring stuff is super tricky and I can certainly imagine regressing without it. E.g. [we have a bug right now](https://github.com/coreos/rpm-ostree/issues/4653) probably related to this.